extension/content_script.js

```diff
@@ -306,6 +304,7 @@ function highlightKeyWords(container, start, end) {
 			var word_start = containerText.indexOf(word, keyword_search_start_pointer);
 			keyword_search_start_pointer = word_start + word.length;
 		};
+		// TODO: Consider optimizing by preprocessing all the keywords and u just add / remove classes.
```

Comment:
so we still have  <mark> els around cos of this.
will be a bigger cl to preprocess so leaving it out.